src/main/java/core/basesyntax/Square.java

```diff
@@ -0,0 +1,23 @@
+package core.basesyntax;
+
+public class Square extends Figure {
+    private final double side;
+    private String color;
```

Comment:
Consider whether the 'color' field should be a class-level variable. Since the 'Figure' class might already contain a 'color' field, you may not need to declare it again in the 'Square' class, unless there's a specific reason to override it.